core/tests/performance_tests/base.py

```diff
@@ -36,9 +44,22 @@ def _record_page_metrics_from_cached_session(self, page_url):
             self.data_fetcher.get_page_metrics_from_cached_session(page_url))
 
     def _record_page_timings_for_url(self, page_url):
-        self.page_metrics = (
-            self.data_fetcher.get_page_timings_for_url(page_url))
+        page_session_metrics = []
+
+        for _ in range(self.SESSION_COUNT):
```

Comment:
Personally, I use xrange almost everywhere, as it is memory efficient. Although, I don't know oppia's recommendation.